orchagent/pfcwdorch.cpp

```diff
@@ -604,13 +809,20 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::doTask(swss::NotificationConsumer
         }
         else
         {
-            throw runtime_error("Unknown PFC WD action");
+            SWSS_LOG_ERROR("Unknown PFC WD action");
         }
     }
     else if (event == "restore")
     {
         if (entry->second.handler != nullptr)
         {
+            SWSS_LOG_NOTICE(
```

Comment:
Why move logs from handler to this orch?

Reply:
Since in big-red-switch mode, we don't want to have log relates to storm detect/restore. Moving log out of handler class provides flexibility for users to input various logs depending on the use case when they create action handler

Reply:
ok